.travis.yml

```diff
@@ -6,7 +6,7 @@ node_js:
 script:
   - npm run build
   - npm run lint
-  - npm run pretest && npm run test:unit
+  - npm run test:noschema # already run schema in build
```

Comment:
It's a lazy form of "We've already run", so run should be correct?  

In any case, I'm not gonna fix it here to avoid conflicts.